Library/Homebrew/dev-cmd/update-license-data.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "commands"
+require "cli/parser"
+require "json"
+require "net/http"
+require "open-uri"
+
+module Homebrew
+  module_function
+
+  SPDX_PATH = (HOMEBREW_LIBRARY_PATH/"data/spdx.json").freeze
+  SPDX_DATA_URL = "https://raw.githubusercontent.com/spdx/license-list-data/HEAD/json/licenses.json"
+
+  def update_license_data_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `update_license_data` <cmd>
+
+         Update SPDX license data in the Homebrew repository.
+      EOS
+      switch "--fail-if-changed",
+             description: "Return a failing status code if current license data's version is different from " \
+                          "the upstream. This can be used to notify CI when the SPDX license data is out of date."
+
+      max_named 0
+    end
+  end
+
+  def update_license_data
+    update_license_data_args.parse
+    ohai "Updating SPDX license data..."
+    spdx_download_result = curl(SPDX_DATA_URL, print_stdout: false)
+    SPDX_PATH.unlink if SPDX_PATH.exist?
+    SPDX_PATH.write(spdx_download_result.stdout)
```

Comment:
```suggestion
    spdx_download_result = curl(SPDX_DATA_URL, print_stdout: false)
    SPDX_PATH.unlink if SPDX_PATH.exist?
    SPDX_PATH.write(spdx_download_result.stdout)
```

Actually sorry, even more stupid me: I totally forgot about `curl_download url, to: filename` that should be able to replace all three lines, I think.